src/intree.js

```diff
@@ -114,6 +123,11 @@ module.exports.setup = function(cfg) {
     if (config.tasks.length > 0) {
       let taskGroupId = config.tasks[0].taskId;
       config.tasks = config.tasks.map((task) => {
+        // Temporary for debugging purposes
+        if (task.task.extra.github.events.some(event => event == 'release')) {
```

Comment:
The depth of the dot-access here worries me. Are we sure we won't have an issue with one of these fields not existing or something?